src/main/java/io/jenkins/plugins/orka/client/OrkaClient.java

```diff
@@ -160,6 +168,11 @@ String delete(String url, String body) throws IOException {
         Request request = this.getAuthenticatedBuilder(url).delete(requestBody).build();
         return executeCall(request);
     }
+    
+    private void setEndpointAndToken(String endpoint, String email, String password) throws IOException {
```

Comment:
This does not bring much value. You are extracting it to remove duplicated code, but 2 lines are ok.
Plus I would call it `initialize` or sth like that, because the next time you add another field the name will need to change again.

To sum up - I would inline this method instead.